content/en/serverless/google_cloud_run/_index.md

```diff
@@ -10,183 +10,508 @@ further_reading:
 
 ## Overview
 
-Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Cloud Run through the [Google Cloud integration][1]. Datadog also provides a solution for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom metrics, and direct log collection.
+Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Cloud Run through the [Google Cloud integration][1]. 
 
-### Prerequisites
+<div class="alert alert-info">To instrument your Google Cloud Run applications with <code>serverless-init</code>, see <a href="/serverless/guide/gcr_serverless_init">Instrument Google Cloud Run with serverless-init</a>.</div>
 
-Make sure you have a [Datadog API Key][6] and are using a programming language [supported by a Datadog tracing library][2].
+## Setup
 
-## Instrument your application
+### Application
 
-You can instrument your application in one of two ways: [Dockerfile](#dockerfile) or [buildpack](#buildpack).
+{{< tabs >}}
+{{% tab "Node.js" %}}
```

Comment:
I think we are missing `ENV NODE_OPTIONS="--require dd-trace/init"` `NODE_OPTIONS` specifies that the dd-trace/init module should be required when the Node.js process starts.